### PR TITLE
plat-ls: Add support for armv8 platform flavours

### PR DESCRIPTION
**Pull request by b49020**
Added support for armv8 platform flavours as follows:
- PLATFORM = ls-ls1043ardb
- PLATFORM = ls-ls1046ardb

Signed-off-by: Sumit Garg <sumit.garg@nxp.com>

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: .travis.yml [hunk 1]
```diff
@@ -216,10 +216,16 @@ script:
   - $make PLATFORM=sprd-sc9860
   - $make PLATFORM=sprd-sc9860 CFG_ARM64_core=y
 
-  # FSL ls1021a
+  # Layerscape ls1021a
   - $make PLATFORM=ls-ls1021atwr
   - $make PLATFORM=ls-ls1021aqds
 
+  # Layerscape ls1043ardb (ARMv8A)
```
Comment (jforissier) [inline .travis.yml]: Minor: the correct (official) spelling is `ARMv8-A`

File: MAINTAINERS.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ for these platforms.
 | Allwinner A80 Board			|`Sun Yangbang <sunny@allwinnertech.com>`|
 | ARM Juno Board			|`Linaro <op-tee@linaro.org>`|
 | Atmel ATSAMA5D2-XULT Board		|`Akshay Bhat <akshay.bhat@timesys.com>`|
-| FSL ls1021a				|`Sumit Garg <sumit.garg@freescale.com>`|
+| FSL ls1021a				|`Sumit Garg <sumit.garg@nxp.com>`|
+| NXP ls1043ardb			|`Sumit Garg <sumit.garg@nxp.com>`|
+| NXP ls1046ardb			|`Sumit Garg <sumit.garg@nxp.com>`|
```
Comment (jforissier) [inline MAINTAINERS.md]: Minor:
I'm a bit confused about naming here. You replaced FSL with Layerscape in `.travis.yml`, and here you're introducing a mix of `FSL` and `NXP`. Should that be `NXP Layerscape ls1021a`, `NXP Layerscape ls1043ardb` etc. maybe?

Comment (b49020) [reply MAINTAINERS.md]: Sure I will fix it.

File: .travis.yml (hunk 1 above)
Comment (b49020) [reply .travis.yml]: I will fix it.

**Pull request merged**